src/components/DetailsOverlay/ButtonRow/index.tsx

```diff
@@ -1,13 +1,52 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
 import Button from "@leafygreen-ui/button";
+import Tooltip from "@leafygreen-ui/tooltip";
 import Icon from "components/Icon";
+import { QueryParams } from "constants/queryParams";
+import { useLogContext } from "context/LogContext";
+import { useQueryParam } from "hooks/useQueryParam";
+import { constructJiraString, copyToClipboard } from "utils/string";
 import { DetailRow } from "../styles";
 
-const ButtonRow: React.FC = () => (
-  <DetailRow>
-    <Button leftGlyph={<Icon glyph="Copy" />}>JIRA</Button>
-    <Button leftGlyph={<Icon glyph="Export" />}>RAW</Button>
-    <Button leftGlyph={<Icon glyph="Export" />}>HTML</Button>
-  </DetailRow>
-);
+const ButtonRow: React.FC = () => {
+  const [hasCopied, setHasCopied] = useState(false);
+
+  const [bookmarks] = useQueryParam<number[]>(QueryParams.Bookmarks, []);
+  const { logLines } = useLogContext();
+  const jiraString = constructJiraString(bookmarks, logLines);
```

Comment:
As a minor perf improvement Maybe we shouldn't generate the string at render time. I think it would make sense to generate it only when the user clicks on the button.